kibana-reports/public/components/main/report_definitions_table.tsx

```diff
@@ -78,7 +33,9 @@ const emptyMessageReportDefinitions = (
         <EuiText>Create a new report definition to get started</EuiText>
         <EuiText>
           To learn more, see{' '}
-          <EuiLink>Get started with Kibana reporting</EuiLink>
+          <EuiLink>
```

Comment:
Some of those EuiLinks don't seem to do anything, is this on purpose?

Reply:
They are placeholder links for now to render the UI- they will be populated once the pages are running (e.g the Reporting documentation page)